Replicated/Replicated.cpp

```diff
@@ -26,6 +26,17 @@ void Replicated::AddBlockToLevel(GameObject *g, GameWorld& world, PrimitiveGameO
         .SetPosition(currentPrimitive->position);
 }
 
+void Replicated::AddSwingingBlock(GameObject* g, GameWorld& world) {
+
+    world.AddGameObject(g, true);
+    auto volume = new AABBVolume(Vector3(10, 10, 10) * 0.5f);
```

Comment:
This should probably be changed from flat values, different swinging objects may require different sizes maybe?